lib/Service/ParticipantService.php

```diff
@@ -1956,14 +1976,14 @@ public function getParticipantUserIdsForCallNotifications(Room $room): array {
 			);
 		}
 
-		$userIds = [];
+		$users = [];
 		$result = $query->executeQuery();
 		while ($row = $result->fetch()) {
-			$userIds[] = $row['actor_id'];
+			$users[$row['actor_id']] = (bool)$row['important'];
```

Comment:
What happens if the user is found twice, once with `important===true` and once not? Is that possible?

Reply:
Unique constraint on DB says no